docs/_articles/en/v3.x/release-information.md

```diff
@@ -12,6 +12,17 @@ We publish the latest Code Analyzer monthly.
 * Run `sfdx plugins:update` and `sfdx plugins:update --help` to update Code Analyzer and help to the latest version.
 * Follow these [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in) to update Code Analyzer
 
+## [v3.10.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.10.0) (03-01-2023)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.10.0...v3.9.0)
+
+### Release Summary
+
+* NEW (CodeAnalyzer): We updated the PMD engine to version 6.54.0.
+* NEW (CodeAnalyzer): We made some updates to the RetireJS vulnerability database.
+* FIX (CodeAnalyzer): We updated the reference to `eslint-recommended` config to point to the updated ESLint dependency. This resolves this issue with new direct or CI/CD installs of Code Analyzer: ```ERROR Cannot find module '/home/runner/.local/share/sfdx/node_modules/eslint/conf/eslint-recommended.js```
+* FIX (GraphEngine): We added a configurable upper limit variable, `SFGE_PATH_EXPANSION_LIMIT`, that lets you define the number of instances of a class in the registry. We also added a new parameter, `--pathexplimit`, that lets you increase or decrease the complexity of Graph Engine scans. These upgrades reduce the number of `OutOfMemory` errors produced. For more information, check out our [documentation](en/v3.x/salesforce-graph-engine/working-with-sfge/#understand-outofmemory-errors).
```

Comment:
`SFGE_PATH_EXPANSION_LIMIT` and `--pathexplimit` are alternatives to do the same thing.
`--pathexplimit` is a parameter on `scanner:run:dfa`. `SFGE_PATH_EXPANSION_LIMIT` is an environment variable, which would probably be used less frequently.

Can we put them together like this?

> We added a new parameter, `--pathexplimit`, for `scanner:run:dfa` command to customize Graph engine run to your codebase's complexity. `SFGE_PATH_EXPANSION_LIMIT` is an alternative environment variable to provide the same value. This upgrade reduces the number of `OutOfMemory` errors produced. For more information, check out our [documentation](en/v3.x/salesforce-graph-engine/working-with-sfge/#understand-outofmemory-errors).